lib/install/management/appliance.go

```diff
@@ -536,12 +536,6 @@ func (d *Dispatcher) createAppliance(conf *config.VirtualContainerHostConfigSpec
 		},
 		Dir: "/home/vicadmin",
 	}
-	if settings.HTTPProxy != nil {
```

Comment:
Will change to this approach following @emlin's comment that VicMachineCreateOptions was added for debug purposes.